packages/tools/devtools/devtools-example/package.json

```diff
@@ -36,12 +36,12 @@
 		"test:jest:coverage": "jest --detectOpenHandles --coverage --ci",
 		"tsc:watch": "tsc --watch",
 		"watch:devtools-core": "tsc --watch --project ../devtools-core/tsconfig.json --preserveWatchOutput",
-		"watch:devtools-view": "tsc --watch --project ../devtools-view/tsconfig.json --preserveWatchOutput",
+		"watch:devtools-view": "tsc --watch --project ../devtools-view/tsconfig.esm.json --preserveWatchOutput",
```

Comment:
Why is this needed?

Reply:
Now that we've migrated to explicit `cjs` and `esm` builds, invoking tsc with our "base" configs spits out errors (currently in the form of a bunch of bogus build output files). Since the example app is `esm`-only, we want to specify the esm build config for devtools-view.

devtools-core doesn't have the same setup (though I'm not sure why), so its config is left alone here for now.